libexec/rc/rc.conf

```diff
@@ -138,6 +138,7 @@ dhclient_flags=""		# Extra flags to pass to dhcp client.
 #dhclient_flags_em0=""		# Extra dhclient flags for em0 only
 background_dhclient="NO"	# Start dhcp client in the background.
 #background_dhclient_em0="YES"	# Start dhcp client on em0 in the background.
+dhclient_arpwait="YES"	# Wait for ARP resolution
```

Comment:
Looks like we need an extra tab to line the comment up with the rest?  Or is that just github being wonky?